site/docs/foundations/characteristics/index.mdx

```diff
@@ -0,0 +1,110 @@
+  --saltCard-background: var(--salt-container-secondary-background);
+  --saltCard-borderColor: var(--salt-container-secondary-borderColor);
+  --saltCard-borderRadius: 10px;
+  --saltCard-borderWidth: 3px;
+}
+```
+
+Note the use of `secondary` container characteristic tokens provided by Salt that weren't used in Card by default, as well as raw values to further emphasize the component. Now, all we have to do to use this secondary variant styling is pass in the `saltCard-secondary` class name we defined to Card (this name has been chosen to follow convention, but could really be anything you like):
+
+Alternatively, we could actually redefine our design to mean that all Card components in our app convey an important message which could be accented. In this case, we'll override with accent characteristics:
+
+```css
+.my-app .saltCard {
+  --saltCard-background: var(--salt-accent-background);
+  --saltCard-borderColor: var(--salt-accent-borderColor);
+  --saltCard-color: var(--salt-accent-foreground);
+}
+```
+
+Note how no class name is needed to be passed in. We have just redefined the Card's styling at a higher specificity, to produce the following styling:
+
+### How do I know which characteristics to apply?
+
+Generally, the characteristics appropriate for the component should be given by the design team. To decide which characteristics to use when styling a new component, it may be useful to ask questions like:
+
+- What can the user do with this component? Can they click it, drag it, etc?
+- What state can this component be in? Can it be focused, disabled, etc?
+- What is the purpose of this component? Will it contain other elements, lead me to a new page, etc?
+- Should this component give feedback? Can it be submitted, does it have a success state, etc?
+
+It is important to note that just because you have assigned a characteristic to a component, does not mean it consumes _all_ of those characteristics. We can freely pick which characteristics we want. A good example of this may be to notice that the `focused` characteristic offers both a border and outline style. From this you would probably want to use an outline if you're setting the outline attribute, and of course border for the border attribute. Yet, there is nothing stopping you from writing `border-style: var(--salt-focused-outlineStyle)` instead.
+
+Think carefully when deciding which characteristics to use. It's up to you to choose which characteristic is most appropriate - for example, should the background color of your component be coming from container, because that's its general purpose, or should it be coming from selectable, since it can be selected? Many characteristics offer the same attribute (like background color), and it's up to you to decide what part of your component you are styling as to which characteristic it should come from. Taking time to consider this will ensure you have the most consistent styling possible across your app.
```

Comment:
```suggestion
It is important to note that just because you have assigned a characteristic to a component, does not mean it consumes _all_ of those characteristics. We can freely pick which characteristics we want. A good example of this may be to notice that the `focused` characteristic offers both a border and outline style. From this you would want to use the outline token if you're setting the outline attribute, and the border token if setting the border attribute.

Think carefully when deciding which characteristics are most appropriate. For example, should the background color of your component be coming from container, because that's its general purpose, or should it be coming from selectable, since it can be selected? Many characteristics host for the same attribute (e.g. background color), and it's important to make the correct decision as to which characteristic your component should adopt the token from. Taking time to consider this will ensure you have the most consistent styling possible across your app, especially when different themes are applied.
```